Source/Core/VideoCommon/PixelShaderGen.cpp

```diff
@@ -372,6 +372,13 @@ void WritePixelShaderCommonHeader(ShaderCode& out, APIType api_type,
             "int3 iround(float3 x) {{ return int3(round(x)); }}\n"
             "int4 iround(float4 x) {{ return int4(round(x)); }}\n\n");
 
+  // GLSL's any() and all() only accept vector types, not scalar types
```

Comment:
Perhaps saying "while HLSL's also accept scalar types" would make it more obvious that this is for compatibility.